chalk-integration/src/interner.rs

```diff
@@ -28,6 +28,12 @@ impl Debug for RawId {
     }
 }
 
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
+pub enum ChalkFnAbi {
+    Rust,
+    C,
```

Comment:
The `system` ABI is also stable. The standard library also uses `rust-intrinsic`, `rust-call`, `platform-intrinsic` and maybe some other ABI's. Maybe use a string instead to be able to represent all ABI's without changes to chalk.

Reply:
This is just the testing harness, the actual ABI type is already an associated type that can be anything.